FitnessApp.Logic/Services/NutrientCategoryService.cs

```diff
@@ -99,6 +105,7 @@ public async Task CreateAsync(NutrientCategoryDto nutrientCategoryDto)
             }
             catch
```

Comment:
in all places
catch(Exception ex) {
        _eventBus.Publish(new LogEvent(Statuses.Fail, "Creation", nutrientCategoryDto.GetType().Name.Replace("Dto", ""), $"Changes was not saved in data base: {ex.Message}"));
}

Reply:
Done